shardingsphere-sql-parser/shardingsphere-sql-parser-binder/src/main/java/org/apache/shardingsphere/sql/parser/binder/metadata/column/ColumnMetaData.java

```diff
@@ -36,4 +37,7 @@
     private final String dataType;
     
     private final boolean primaryKey;
+    
+    @Setter
+    private boolean isCaseSensitive;
```

Comment:
Please consider change `isCaseSensitive` to `caseSensitive`

Reply:
Fixed.